BUILD

```diff
@@ -40,3 +47,15 @@ kt_compiler_plugin(
         "@maven//:androidx_compose_compiler_compiler",
     ],
 )
+
+# Add missing 'sun.misc' files to coroutines artifact
+# Used in 'override_targets' by referencing @//:kotlinx_coroutines_core_jvm
+kt_jvm_import(
+    name = "kotlinx_coroutines_core_jvm",
+    jars = ["@maven_secondary//:v1/https/repo1.maven.org/maven2/org/jetbrains/kotlinx/kotlinx-coroutines-core-jvm/1.5.1/kotlinx-coroutines-core-jvm-1.5.1.jar"],
```

Comment:
You also need to pass the source jar otherwise it can break the IntelliJ Bazel Plug-in aspect.

Reply:
Added `srcjar`